### ISSUE TITLE
[DOC]: dfencoder's more mainstream cousin is...?

### ISSUE THREAD

**Issue by nyck33** [nv-morpheus/Morpheus#1228]
### How would you describe the priority of this documentation request

Low (would be nice)

### Describe the future/missing documentation

https://github.com/AlliedToasters/dfencoder

It seems fairly minor and as the author states is a personal project so development is slow.  

Do you have any recommendations of more mainstream libraries?

https://keras.io/examples/timeseries/timeseries_anomaly_detection/

https://www.tensorflow.org/tutorials/generative/autoencoder

What was the decision behind using such an obscure library for autoencoders?

### Where have you looked?

see above

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct
- [X] I have searched the [open documentation issues](https://github.com/nv-morpheus/Morpheus/issues?q=is%3Aopen+is%3Aissue+label%3Adoc) and have found no duplicates for this bug report

**Comment by jarmak-nv** [comment]
Hi @nyck33!
  
  Thanks for submitting this issue - our team has been notified and we'll get back to you as soon as we can!
  In the mean time, feel free to add any relevant information to this issue.
